docs/explanations/fix_scenarios.rst

```diff
@@ -1,23 +1,20 @@
-Use ``pro fix`` to solve a CVE/USN
-**********************************
+.. _pro-fix-howto:
 
-The Ubuntu Pro Client (``pro``) can be used to inspect and resolve
-`Common Vulnerabilities and Exposures <cve_>`_ (CVEs) and
-`Ubuntu Security Notices <usn_>`_ (USNs) on your machine.
+Scenarios encountered using ``pro fix`` to solve a CVE/USN
+**********************************************************
 
-Every CVE/USN is fixed by trying to upgrade all of the affected packages
-described by the CVE or USN. Sometimes, the package fixes can only be applied
-if an Ubuntu Pro service is already enabled on your machine.
+.. Into "what is pro fix" shared with the related tutorial
+.. include:: ../includes/pro-fix-intro.txt
 
-In this tutorial, we will introduce the ``pro fix`` command and test some
-common scenarios you may encounter.
+This page will go a bit deeper and after introducing the ``pro fix``
+command it will go in more details about the different scenarios you may
+encounter using ``pro fix``.
 
-.. Why we use Multipass + command to install it
-.. include:: ./common/install-multipass.txt
-
-.. Commands for launching and updating a Xenial VM
-.. include:: ./common/create-vm.txt
+.. note::
 
+   If instead you look for a simpler guided tutorial to get started with
+   ``pro fix`` please start at
+   :ref:`Use pro fix to solve a CVE/USN <pro-fix-tutorial>`.
```

Comment:
This document has sentences such as:

`We will choose to fix a CVE that does not affect the VM`,

But we only mention the VM on the tutorial. Should we mention that we expect the user to have a similar setup to follow this how-to-guide ?

Reply:
done